azure-pipelines/integrationtests.yml

```diff
@@ -63,26 +63,27 @@ steps:
     env:
       sqlOnPrem_password: '$(sqlOnPrem_password)'
 
-  - task: AzureKeyVault@1
-    displayName: 'Azure Key Vault: SqlToolsSecretStore'
-    inputs:
-      azureSubscription: 'STS Pipelines' # this is the name of the service connection, which be found in mssqltools/CrossPlatBuildScripts ADO under Project Settings -> Service Connections
-      KeyVaultName: SqlToolsSecretStore
-      SecretsFilter: 'sqltools-backup-url-tests-blobcontaineruri,sqltools-backup-url-tests-storageaccountname,sqltools-backup-url-tests-storageaccountkey'
+  # Uncomment tasks that use secrets while we get context on managed identity
+  # - task: AzureKeyVault@1

```

Comment:
Actually, let's completely delete this `AzureKeyVault@1` step instead of just commenting it out.  But still leave the "Running integration tests" step below commented-out (instead of deleted).